### PR TITLE
Openoffice pt updated

### PR DESCRIPTION
**Pull request by alexandregz**

### CODE REVIEW

File: Casks/openoffice-pt.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'openoffice-pt' do
+  version '4.1.1'
+  sha256 'b224515d21bc8826d92aba6cc18f044883b12ee6e401e7db0b6feb67f76aeedb'
+
+  url "http://downloads.sourceforge.net/project/openofficeorg.mirror/#{version}/binaries/pt/Apache_OpenOffice_#{version}_MacOS_x86-64_install_pt.dmg"
+  homepage 'http://www.openoffice.org/pt/download/'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/openoffice-pt.rb]: Should be `:apache`.

**Comment by vitorgalvao** [comment]
Please add `name`. Follow [the main cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/openoffice.rb)

File: Casks/openoffice-pt.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'openoffice-pt' do
+  version '4.1.1'
+  sha256 'b224515d21bc8826d92aba6cc18f044883b12ee6e401e7db0b6feb67f76aeedb'
+
+  url "http://downloads.sourceforge.net/project/openofficeorg.mirror/#{version}/binaries/pt/Apache_OpenOffice_#{version}_MacOS_x86-64_install_pt.dmg"
```
Comment (vitorgalvao) [inline Casks/openoffice-pt.rb]: It [needs a comment](https://github.com/caskroom/homebrew-cask/blob/465afa9e32a0b841c02bd71031d54c5fbcc68fbf/Casks/openoffice.rb#L5) above.

**Comment by alexandregz** [comment]
I made changes to formula, thx for the infor about that!

**Comment by vitorgalvao** [comment]
Looks good, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
Looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by alexandregz** [comment]
openoffice formulas or another ones too?

**Comment by vitorgalvao** [comment]
Every submission should be squashed to the least amount of logical steps (only one is needed for new submissions).

**Comment by alexandregz** [comment]
rebased

**Comment by vitorgalvao** [comment]
There are still two commits. It needs to be one.

**Comment by alexandregz** [comment]
Sorry, I don't understand you, I make the squash now.

**Comment by vitorgalvao** [comment]
Now it looks great. Thank you for all the work.

**Comment by alexandregz** [comment]
thx for your patience :smiley:

**Pull request merged**